src/markdown-pages/build-apps/permission-manage-apps.mdx

```diff
@@ -15,40 +15,31 @@ tags:
 
 <Intro>
 
-When you create an app, you'll likely want to share it. From New Relic One's **Apps** page, you can subscribe to apps you create, publish, and deploy, and to other publicly available apps.
-
-You must have the **Nerdpack manager** role to subcribe accounts to apps. Read on to learn about permissions and versions.
+Details about permissions required for creating and managing [New Relic One applications](https://developer.newrelic.com/build-apps).
 
 </Intro>
 
-## Permissions for managing applications
-
-The Nerdpack manager role is a [New Relic add-on role](https://docs.newrelic.com/docs/accounts/accounts/roles-permissions/add-roles-permissions). When you create a Nerdpack, you have the Nerdpack manager role for handling that Nerdpack. New Relic account administrators have the Nerdpack manager role automatically, and can subscribe their accounts to available Nerdpacks.
-
-User permissions vary depending on which pricing plan you are on.
-
-### New Relic One pricing plan
-
-For accounts with New Relic One pricing, there are permissions differences for basic users and full users:
+There are several factors affecting your New Relic One app permissions. 
 
-**Full users** have the Nerdpack manager role and have full capabilities for creating and managing New Relic One applications, as well as accessing all types of applications in the New Relic One catalog.
+## Basic user restrictions 
 
-A **basic user** can develop and view their own local New Relic One apps, but they ***cannot***:
+The most important permissions factor is [user type](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#user-type): 
 
-- Subscribe other users to apps they’ve created.
-- Access or manage apps in the New Relic One catalog.
-- Access apps in the entity explorer sidebar.
+* **Basic users** can only develop New Relic One apps locally. They can’t publish or subscribe to an app, whether it's deployed in their account, available in [the public catalog](https://opensource.newrelic.com/nerdpacks/), or one they've built. 
```

Comment:
```suggestion
* **Basic users** can only develop New Relic One apps locally. They can’t publish or subscribe to an app, whether it's deployed in their account, available in the [New Relic One catalog](https://opensource.newrelic.com/nerdpacks/), or one they've built. 
```

Reply:
Let's use the correct product term here...

Also I'm not clear what this means?

```They can’t publish or subscribe to an app, whether it's deployed in their account, available in the New Relic One catalog```

What are we trying to communicate here?

Reply:
@jpvajda I'm not sure what you mean by "correct product term" sorry. Could you clarify on that. 

We are trying to communicate that basic users can't actually publish a NR1 app, or subscribe an account/users to it (essentially they can't do higher level mgmt things with it), and that is true no matter if we're talking a NR1 app built for their NR account/org, or an app that's in our catalog (publicly available), or one they themselves have built and are trying to share in their org. Basically we're trying to clarify that this rule applies no matter the source of the app, because there has been confusion around 'well maybe this is true for NR1 app catalog but maybe I can publish it for my organization', things like that.

Reply:
A lot of this terminology is confusing in general, not just in this document 😄 Technically, a user publishes a Nerdpack to the same New Relic One catalog that contains the publicly available, first-party New Relic apps. We just filter the apps that users can see. So, distinguishing between those that are "deployed in their account" and those that are "available in the New Relic One catalog" is slightly misleading.

I think something like this might clear that up:

> They can't publish or subscribe to any apps or custom visualizations, including their own.

Reply:
Oh sorry, I didn't see the "catalog"-related suggested change, makes sense now.